docs/io/fits/index.rst

```diff
@@ -47,8 +47,8 @@ Once the `astropy.io.fits` package is loaded using the standard convention
 
     >>> hdul = fits.open(fits_image_filename)
 
-Note : The :func:`get_testdata_filepath` function is only used by tests to get their data.
-Please only use :func:`fits.open` along with the path to the appropraite FITS data file
+Note : The ~astropy.io.fits.util.get_testdata_filepath function is only used by tests to get their data.
```

Comment:
Not exactly. Proper linking syntax would be
```python
:func:`~astropy.io.fits.util.get_testdata_filepath`
```

Reply:
To be sure that is correct, you could try running `python setup.py build_docs` and see if there are any errors. Please refer to http://docs.astropy.org/en/stable/development/docguide.html when in doubt. Thanks!

Reply:
Thanks for the guidelines!

Reply:
Oh, indeed the backticks are important, I didn't pay attention that github markdown rendered them out of my previous comment. The `:func:` prefix though is optional.

Reply:
Actually, I'm taking back everything I said above. Please use double backticks around the word `get_testdata_filepath`. It's not in the API reference, so no linking will work.
Sorry about the extra rounds.